drivers/crypto/crypto_mtls_shim.c

```diff
@@ -103,36 +104,55 @@ int mtls_ecb_decrypt(struct cipher_ctx *ctx, struct cipher_pkt *pkt)
 
 int mtls_cbc_encrypt(struct cipher_ctx *ctx, struct cipher_pkt *pkt, u8_t *iv)
 {
-	int ret;
+	int ret, iv_bytes;
+	u8_t *p_iv, iv_loc[16];
 	mbedtls_aes_context *cbc_ctx = MTLS_GET_CTX(ctx, aes);
 
-	/* Prefix IV to ciphertext */
```

Comment:
Can we preserve this comment, moving to the appropriate branch, perhaps extending as "Prefix IV to ciphertext, which is default behavior of Zephyr crypto API, unless CAP_NO_IV_PREFIX is requested."

Reply:
Done.

Reply:
Thanks. Looking forward to a push to approve it then. Let's see if that random CI failure will go away too...